static/js/src/advantage/offers/components/Offer/Offer.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { ActionButton, Card } from "@canonical/react-components";
+import { currencyFormatter } from "advantage/react/utils";
+import { Offer as OfferType, Item } from "../../types";
+
+type Props = {
+  offer: OfferType;
+};
+
+const Offer = ({ offer }: Props) => {
+  return (
+    <Card data-testid="offer-card">
+      <div className="advantage-offer-row">
+        <p className="p-text--x-small-capitalised contract-item">
+          Contract item
+        </p>
+        <p className="p-text--x-small-capitalised allowance">Allowance</p>
+        <p className="p-text--x-small-capitalised price">Price</p>
+      </div>
+      <hr />
+      {offer.items.map((item: Item) => {
+        return (
+          <div key={item.id} className="advantage-offer-row">
+            <p className="contract-item">
+              <strong>{item.name}</strong>
+            </p>
+            <p className="allowance">{item.allowance ?? 0}</p>
+            <p className="price">
+              {currencyFormatter.format(item.price / 100)}
+            </p>
+          </div>
+        );
+      })}
+      <div className="advantage-offer-row">
+        <p className="p-text--x-small-capitalised allowance">
+          Total before taxes
+        </p>
+        <p className="price">{currencyFormatter.format(offer.total / 100)}</p>
+      </div>
+      <div
+        style={{ width: "100%", display: "flex", justifyContent: "flex-end" }}
```

Comment:
Can you use vanilla utilities instead of inline styles?

Reply:
I tried first with Vanilla utilites but couldn't get it to work the way I wanted. Could you point me in the right direction?

Reply:
I think I got it!